### ISSUE TITLE
wrong pagenagviate 

### ISSUE THREAD

**Issue by GuangchuangYu** [parsiya/Hugo-Octopress#14]
thanks for this great theme.

I build a blog with `baseurl = "http://guangchuangyu.github.io/cn/"`, all the links (tags, categories, etc) are OK except paginator. Paginator link e.g. page2 to `http://guangchuangyu.github.io/page/2` instead of `http://guangchuangyu.github.io/cn/page/2`

**Comment by parsiya** [comment]
Good catch, my baseurl points to the root of my website so I have never encountered this.

Seems like this is a Hugo issue. https://github.com/spf13/hugo/issues/1252

For now please set this in the config file:

    CanonifyUrls = "true"

Until I go ahead and fix the pagination template.

**Comment by GuangchuangYu** [comment]
thanks for your reply.

the `CanonifyUrls = "true"` works.

**Comment by kulla** [comment]
Note: I have written a [fix](https://github.com/spf13/hugo/pull/2403) for [spf13/hugo#1252](https://github.com/spf13/hugo/issues/1252). It might help as well (it worked for me).

**Comment by parsiya** [comment]
Hi Stephan. Thanks for helping with the code. I will look into your commit and see if I can manually do the same in the theme code.
